resources/views/community/components/user/progression-status/legend.blade.php

```diff
@@ -0,0 +1,41 @@
+@props([
+    'totalBeatenHardcoreCount' => 0,
+    'totalBeatenSoftcoreCount' => 0,
+    'totalCompletedCount' => 0,
+    'totalMasteredCount' => 0,
+])
+
+<div class="{{ ($totalCompletedCount || $totalBeatenSoftcoreCount) ? 'grid grid-cols-2' : 'flex' }} sm:flex gap-x-4 [&>div]:flex [&>div]:items-center [&>div]:gap-x-1">
+    <div class="order-1">
+        <div class="rounded-full w-2 h-2 bg-zinc-500"></div>
+        <p class="text-zinc-500 font-bold text-xs">Unfinished</p>
+    </div>
```

Comment:
minor: Unfinished header has a disc, rows do not.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/8a060c90-ce10-4c62-9bb9-955030b3c719)

Reply:
Disc removed for Unfinished in latest.